# Goal
string object_name
---
# Result
bool success
string message
---
# Feedback
string status